.github/workflows/chromatic.yml

```diff
@@ -0,0 +1,18 @@
+# Publish Storybooks builds on Chromatic
+
+name: "Chromatic"
+
+on: push
+
+jobs:
+  chromatic:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install & build
+        run: |
+          yarn install --frozen-lockfile
+          yarn build
+      - name: Publish to Chromatic
```

Comment:
env: |
 secrets: ${{ github.secrets }}
 ...
 smth like this
 instead of exposion of chromatic tokens